pkg/auth/auth_handler.go

```diff
@@ -179,26 +179,35 @@ func (a *AuthHandler) AuthUserAuth(token string) (interface{}, error) {
 		return nil, err
 	}
 
+	if payload.Username == "" {
+		a.log.Error("Missing username in token")
+		return nil, fmt.Errorf("Missing username in token")
```

Comment:
please use errors package instead of fmt, errors have a stuck trace that we can use later.

Reply:
It's actually not related to this PR, I only moved the existing lines... But sure, will change.